packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -18,72 +18,15 @@ import React from 'react';
 import styled from 'styled-components';
 import { Cell } from 'design/DataTable';
 import Icon, * as Icons from 'design/Icon/Icon';
-import { CodeEnum } from 'teleport/services/audit/types';
-import { Event } from 'teleport/services/audit';
+import { Event, eventTypes as events } from 'teleport/services/audit';
```

Comment:
I found myself getting lost navigating between Events, events, eventTypes _as events_, Typescript event types, etc. `EventType` sounds as a typescript construct but it's not.